civictechprojects/helpers/context_preload.py

```diff
@@ -144,6 +172,29 @@ def terms_preload(context, request):
     context['description'] = 'Terms of Use page'
     return context
 
+def sign_up_preload(context, request):
+    context = default_preload(context, request)
+    context['title'] = 'Sign Up | DemocracyLab'
+    context['description'] = 'Sign up'
+    return context
+
+def contact_us_preload(context, request):
+    context = default_preload(context, request)
+    context['title'] = 'Contact Us | DemocracyLab'
+    context['description'] = 'Contact information for DemocracyLab.'
+    return context
+
+def reset_password_preload(context, request):
+    context = default_preload(context, request)
+    context['title'] = 'Reset Password | DemocracyLab'
+    context['description'] = 'Reset password page'
+    return context
+
+def press_preload(context, request):
+    context = default_preload(context, request)
+    context['title'] = 'News Feed | DemocracyLab'
+    context['description'] =  "Read external articles and blog posts describing DemocracyLab's work."
+    return context
```

Comment:
The press page has been deleted, is there a reason we need this? I think it can be removed unless there's a technical reason to keep it.

Reply:
> The press page has been deleted, is there a reason we need this? I think it can be removed unless there's a technical reason to keep it.

Hello Peter, thanks for reviewing the code and giving suggestions. It should be fine to delete the press page, I didn't know that the press page was deleted